packages/driver/src/dom/elements.js

```diff
@@ -494,6 +494,28 @@ const isTextLike = function ($el) {
   ])
 }
 
+const blocksImplicitSubmission = function ($el) {
```

Comment:
The method name here is confusing because it's actually describing the **exact opposite** of what it does.

These inputs **do not block** implicit submission - they actually **enable** implicit form submission.

For the purposes of `elements.js` - this method should actually be about the input types themselves - it's up to the higher order commands to figure out whether or not these elements **would** in fact implicitly submit the form due to their `length`.

I had to reread your code many times to figure out what it's actually doing because of the name of this method.